src/rust/broker/src/server.rs

```diff
@@ -48,6 +51,20 @@ pub async fn server(tx: Sender<EmojicoinDbEvent>, port: u16) -> Result<(), std::
     let listener = tokio::net::TcpListener::bind(&format!("0.0.0.0:{port}"))
         .await
         .unwrap();
+
+    if cfg!(all(feature = "ws", feature = "sse")) {
+        info!("Starting web server with ws and sse.");
+    }
+    if cfg!(all(feature = "ws", not(feature = "sse"))) {
+        info!("Starting web server with ws.");
+    }
+    if cfg!(all(not(feature = "ws"), feature = "sse")) {
+        info!("Starting web server with sse.");
+    }
+    if cfg!(all(not(feature = "ws"), not(feature = "sse"))) {
+        warn!("Starting web server with no endpoints.");
+    }
+
```

Comment:
Can we put this in a match statement or is it not possible because it's a macro

Reply:
I don't think this is possible. Each on the `cfg!(...)` will resolve to either `true` or `false` at compile time.